api/config/default.json

```diff
@@ -3,7 +3,8 @@
     "port": 3000,
     "loggerLevel": [
       "error",
-      "warn"
+      "warn",
+      "log"
```

Comment:
I believe the default logging level should include this value, as it shows to the terminal all the logger instances present on the code.

Without that value the log shows nothing about i.e map generation logs or service error traces. (It does show endpoint error handlers)

Let me know if you still want to change this

Reply:
IMO the primary goal of logging in our applications should be about making errors or potentially invalid/suspicious activities visible. log/verbose/info/debug logging levels should be an opt-in, because they are only useful in a very limited context - otherwise they will simply produce too much noise for anything to be useful.

More importantly: if you are dependent on logs, you may not be using a debugger properly, which IMO is a much better tool for debugging than an actual logger.

Anyways, this is just an opinion, let me know what you think of it

Reply:
IMO the end goal of having logger instances that are not logging error traces is that we actually want to know whats going on in a staging, prod envs, I believe that why we are logging map generations, tmp table deletions, file imports and more stuff, but this is also my junior opinion

Changed back

Reply:
It's not about junior or senior opinions, this is mostly a matter of personal preference, as well as having an idea of how this will be used. - which then informs a technical discussion ;)